lib/internal/fs/watchers.js

```diff
@@ -34,8 +34,20 @@ function StatWatcher(bigint) {
   EventEmitter.call(this);
 
   this._handle = null;
-  this[kOldStatus] = -1;
-  this[kUseBigint] = bigint;
+  Object.defineProperties(this, {
+    [kOldStatus]: {
+      value: -1,
+      configurable: true,
+      writable: true,
+      enumerable: false
+    },
+    [kUseBigint]: {
+      value: bigint,
+      configurable: true,
+      writable: true,
+      enumerable: false
+    }
```

Comment:
I guess my main issue with these kinds of simple properties that are now hidden is that there’s no way to tell their state from the default `util.inspect()` output…

Reply:
Well... I am not really sure what to say here. The whole reason for this PR to do exactly that: hide these properties from regular inspection while still keeping the possible to inspect them if done explicitly by using `showHidden: true`.